tests/tests.toml

```diff
@@ -370,6 +370,16 @@ filename = "../integration_tests/print_01.py"
 c = true
 cpp = true
 
+[[test]]
+filename = "../integration_tests/generics_01.py"
+asr = true
+llvm = true
```

Comment:
I don't think we need to test llvm, since we are not modifying anything in the backend. Rather, let's add an integration test, to ensure it builds and runs.

Reply:
Okay got it. I've already added the integration test.